.github/workflows/ISSUE_TEMPLATE/config.yaml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Questions
+    url: https://scenic-lang.readthedocs.io/
```

Comment:
One thing to note is that this URL will change to the forum once it has been set up. Do we want to configure something temporary for now like a Google Form?

Reply:
Agreed that this will be email!